data/en/1_0001.json

```diff
@@ -4714,7 +4724,8 @@
     "messages": {
         "NUMBER_TOO_LARGE": "The maximum value allowed is 9999999999. Please correct your answer.",
         "NUMBER_TOO_SMALL": "The value cannot be negative. Please correct your answer.",
-        "INVALID_NUMBER": "The value you have entered is invalid. Please enter a numeric value."
+        "INVALID_NUMBER": "The value you have entered is invalid. Please enter a numeric value.",
+        "INVALID_DECIMAL": "Please enter a number to 2 decimal places."
```

Comment:
If we remove the default pence allowed for currency values then we also need to remove the INVALID_DECIMAL error message.